src/contracts/pods/EigenPod.sol

```diff
@@ -428,6 +427,38 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         _sendETH(recipient, amountWei);
     }
 
+
+    /*******************************************************************************
+                    EXTERNAL FUNCTIONS CALLABLE BY PERMISSIONED SERVICES
+    *******************************************************************************/
+
+    /// @notice Called by the EigenPodManager to fulfill a partial withdrawal proof request
+    function fulfillPartialWithdrawalProofRequest(
+        IEigenPodManager.Journal calldata journal,
+        uint64 feeGwei,
+        address feeRecipient
+    ) external onlyEigenPodManager onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_WITHDRAWAL) {
+
+        require(journal.mostRecentWithdrawalTimestamp == mostRecentWithdrawalTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: proven mostRecentWithdrawalTimestamp must match mostRecentWithdrawalTimestamp in the EigenPod");
+        require(mostRecentWithdrawalTimestamp < journal.endTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: mostRecentWithdrawalTimestamp must precede endTimestamp");
+
+
+        uint256 provenPartialWithdrawalSumGwei = journal.provenPartialWithdrawalSumGwei;
+        require(sumOfPartialWithdrawalsClaimedGwei <= provenPartialWithdrawalSumGwei - feeGwei, "EigenPod.fulfillPartialWithdrawalProofRequest: sumOfPartialWithdrawalsClaimedGwei must be less than or equal to provenPartialWithdrawalSumGwei + feeGwei");
+        provenPartialWithdrawalSumGwei -= feeGwei;
+
+
+        // subtract an partial withdrawals that may have been claimed via merkle proofs
+        provenPartialWithdrawalSumGwei -= sumOfPartialWithdrawalsClaimedGwei;
```

Comment:
rename `sumOfPartialWithdrawalsClaimedGwei` to `sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei` or someth for clarity pls